### PR TITLE
feat: add linux arm64 build

### PR DESCRIPTION
**Pull request by gengjiawen**
Fix https://github.com/napi-bindings/quickjs-build/issues/9

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,80 +3,90 @@ cmake_minimum_required(VERSION 3.16)
...
-  COMMAND qjsc -c -o ${CMAKE_CURRENT_BINARY_DIR}/repl.c -m
-          ${CMAKE_CURRENT_SOURCE_DIR}/src/repl.js
-  COMMAND qjsc -c -o ${CMAKE_CURRENT_BINARY_DIR}/qjscalc.c -m
-          ${CMAKE_CURRENT_SOURCE_DIR}/src/qjscalc.js
-  DEPENDS qjsc)
+if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "aarch64")
+  add_custom_command(
+    OUTPUT repl.c qjscalc.c
+    COMMAND
+      qemu-aarch64 -L /usr/aarch64-linux-gnu/ qjsc -c -o
```
Comment (gengjiawen) [inline CMakeLists.txt:65]: Not sure there is better way, I try to set `qemu-aarch64 -L /usr/aarch64-linux-gnu/ qjsc` as a variable to reuse the logic, but build failed.

**Comment by gengjiawen** [comment]
@NickNaso PTAL.

**Comment by gengjiawen** [comment]
ping

**Comment by NickNaso** [comment]
> ping

I'm starting to review sorry if I'm doing this only now.

File: CMakeLists.txt (hunk 1 above)
Comment (NickNaso) [reply CMakeLists.txt:65]: I don't know a better solution it's ok for me.

**Review by NickNaso — APPROVED** [review]
I think that it's all good.

**Pull request merged**